src/helpers.ts

```diff
@@ -0,0 +1,39 @@
+export function getNextFrame(
+  current: number,
+  count: number,
+  frameSize: number,
+  step: number,
+  infinite: boolean,
+) {
```

Comment:
```suggestion
): number {
```
Better to add return type